ex3.html

```diff
@@ -0,0 +1,558 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="ex1_bod_css.css">
+	<link rel="stylesheet" type="text/css" href="ex2_header_css.css">
+	<link rel="stylesheet" type="text/css" href="ex3_bod_css.css">
+	<link rel="stylesheet" type="text/css" href="buttons.css">
+</head>
+<body>
+	<section>
+		<header class="header"> 
+			<img src="logo.jpg" style="float:left; width:200px;height:40px; margin-left: 3%">
+			<h2><a href="ex2.html" target="_self" class="button2" style="margin-left: 7%; margin-right: 0.1%;">Evaluations</a></h2>
+			<h2><a href="ex3.html" target="_self" class="button1"> New Evaluation</a></h2>
+			<h4><a href="ex1.html" target="_top" style="float: right; margin-right: 2%; font-size: 150%; text-decoration: none;">Logout</a></h4>
+		</header>
+	</section>
+
+	<section>
+		<div id="dm">
+			<form action="/action_page.php">
+			<input class="textbox" type="text" placeholder="Candidate" name="CandName">
+				<input class="textbox" type="text" placeholder="Interviewer" name="IntervName">
+				<input class="textbox" type="date" name="ddate">
+
+
+				<style type="text/css">
```

Comment:
Any particular reason why this is embedded in the DOM instead of loaded externally?